css/css-grid/alignment/grid-block-axis-alignment-auto-margins-001.html

```diff
@@ -35,13 +35,16 @@
     <div id="item2"></div>
 </div>
 <script>
+document.fonts.ready.then(() => {
     item1.setAttribute("data-offset-y", "75");
     item2.setAttribute("data-offset-y", "300");
-    checkLayout('#grid');
+    checkLayout('#grid', false);
 
     grid.style.height = "300px";
 
     item1.setAttribute("data-offset-y", "35");
     item2.setAttribute("data-offset-y", "160");
-    checkLayout('#grid');
+
+    checkLayout('#grid', true);
```

Comment:
Nit: You don't need to pass "true" on the last call, but it's fine if you want to be explicit here. Feel free to keep it or remove it as you wish.